core/state/snapshot/generate_test.go

```diff
@@ -406,7 +406,7 @@ func TestGenerateCorruptAccountTrie(t *testing.T) {
 		// Snapshot generation succeeded
 		t.Errorf("Snapshot generated against corrupt account trie")
 
-	case <-time.After(250 * time.Millisecond):
+	case <-time.After(3 * time.Second):
```

Comment:
This is the expected case, which means this test will always take three seconds instead of `250ms`. For the cases where the failure is expected, perhaps let's keep it lower (`1s`?) and increase it to `3s` only in the cases where we expect it to actually exit via `<-snap.genPending` and only use the timeout when the test fails

Reply:
Yes. I change it to 1 second.